hs_core/views/utils.py

```diff
@@ -93,6 +106,158 @@ def upload_from_irods(username, password, host, port, zone, irods_fnames, res_fi
     irods_storage.delete_user_session()
 
 
+def validate_url(url):
+    """
+    Validate URL
+    :param url: input url to be validated
+    :return: [True, ''] if url is valid,[False, 'error message'] if url is not valid
+    """
+    # validate url's syntax is valid
+    error_message = "The URL that you entered is not valid. Please enter a valid URL."
+    try:
+        validator = URLValidator(schemes=('http', 'https', 'ftp', 'ftps'))
+        validator(url)
+    except ValidationError:
+        return False, error_message
```

Comment:
I think including the accepted protocols would be helpful in the error message.

Reply:
@sblack-usu This error message is suggested by @horsburgh, i.e., ```The URL that you entered is not valid. Please enter a valid URL.``` @horsburgh What you do think of @sblack-usu 's comment on including the accepted protocols in the error message?

Reply:
@sblack-usu @horsburgh Have updated the validation user message to include the protocols as @horsburgh indicated above.